BinaryOptionsToolsV2/BinaryOptionsToolsV2/tracing.py

```
@@ -5,24 +5,27 @@
+
 class LogSubscription:
     def __init__(self, subscription):
         self.subscription = subscription
-        
+
     def __aiter__(self):
         return self
-        
+
     async def __anext__(self):
         return json.loads(await anext(self.subscription))
```

Ruff (0.13.1)

Undefined name `anext`. Consider specifying `requires-python = ">= 3.10"` or `tool.ruff.target-version = "py310"` in your `pyproject.toml` file.

(F821)